docs/developer-docs/latest/development/snippets/custom-field-plugin-optional.md

```diff
@@ -0,0 +1 @@
+If omitted, the custom field is registered within the `global` namespace and is only accessible to the Strapi application it was created for.
```

Comment:
So if a customField is registered in a Strapi application it will only be available in that Strapi application, the developers doesn't have to provide a `pluginId` but technically they could if they wanted the custom field to live in a specific namespace. 

A plugin on the other hand could also omit this key and the customField would still be usable in many Strapi applications since it can be installed as a plugin, but it would live in the `global::` namespace

Reply:
>A plugin on the other hand could also omit this key and the customField would still be usable in many Strapi applications since it can be installed as a plugin, but it would live in the global:: namespace

I think that's mainly because we haven't found a way to prevent this, but it's not something we want to encourage

Reply:
Thanks for the explanation. Do you think we could keep this line in the documentation as-is?

Reply:
We should probably avoid mentioning this tbh, I agree with Remi here in that we want to avoid it but we can't physically prevent it however if/when we can then we will.

It will at least hope to cut down on the breaking changes in the future by leaving it semi-undocumented.